script/encrypt-heroku-auth-token

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+function cleanup_at_exit {
+  # remove temporary key file
+  rm tmp.pem
+}
+
+trap cleanup_at_exit EXIT
+
+# Get the repo name from the `origin` git remote.
+export REPO=$(git remote get-url origin | \
+    sed 's/^.*[\/:]\([^\/:]*\)\/\([^\/]*\)$/\1\/\2/' | \
+    sed 's/^\(.*\)\.git$/\1/'
+)
+
+# Get the Travis public key for this repo.
+export TRAVIS_KEY=$(curl --silent https://api.travis-ci.org/repos/$REPO/key)
+
+# Save the key to a file.
+node -e "console.log($TRAVIS_KEY['key'])" > tmp.pem
```

Comment:
Maybe we should gitignore this `tmp.pem` file, just in case the script fails to exit properly (invoking the trap).

Reply:
I think it would only stick around if 1) it was created and 2) the script exited in such a way that the `trap` did not execute. I don't know under what conditions that can happen, but I would expect that they would be unlikely. That being said, I have no objection to adding `tmp.pem` to the `.gitignore`.

Reply:
Since it's a security risk I am in favor of the belt-and-suspenders approach.

Reply:
As per our offline convo, since this a public key it's not a big deal. If it was a security risk I'd advocate the belt-and-suspenders approach, but in this case I'd not bother.